terraform/modules/k8s/vault/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "namespace" {
+  description = "Vault namespace"
+  default     = "vault"
+}
+
+variable "vault_pvc_storage_class" {
+  default     = "standard"
```

Comment:
Did you checked it on Azure AKS? As far as I remember, we have to use [`default` storage class](https://docs.microsoft.com/en-us/azure/aks/azure-disks-dynamic-pv#built-in-storage-classes).

Reply:
Good catching! Fixed. I will retest on azure.